src/plugins/legacy/reformat/medCropToolBox.cpp

```diff
@@ -445,36 +459,40 @@ void medCropToolBoxPrivate::applyInverseOrientationMatrix(double *worldPointIn,
     std::copy(homogeneousVector, homogeneousVector + 3, worldPointOut);
 }
 
-void medCropToolBoxPrivate::generateOutput()
+bool medCropToolBox::generateOutput()
 {
+    bool res = medAbstractProcessLegacy::SUCCESS;
```

Comment:
Error codes should not be used with booleans. The type should be `int` or `medAbstractProcessLegacy::DataError`.